baselines/Google Drive and Docs Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -393,7 +393,7 @@ To configure the settings for add-ons:
 
 ## 6. Drive for Desktop
 
-This section covers that Google Drive for Desktop, if not disabled entirely, should only be allowed on authorized devices.
+This section covers the implications of not disabling Drive for Desktop and considerations around authorized device use.
```

Comment:
This wording feels a little awkward. Also, it might be more useful to use this space to describe what Drive for Desktop is, as not all users will be familiar with it.

How about something like this?

```suggestion
This section addresses Drive for Desktop, a feature that enables users to interact with their Drive files directly through their desktop's file system, rather than through the browser.
```